internal/saucecloud/cloud.go

```diff
@@ -140,7 +139,14 @@ func (r *CloudRunner) runJob(opts job.StartOptions) (j job.Job, interrupted bool
 	}
 
 	jobDetailsPage := fmt.Sprintf("%s/tests/%s", r.Region.AppBaseURL(), id)
-	log.Info().Str("suite", opts.Suite).Str("url", jobDetailsPage).Msg("Suite started.")
+	l := log.Info().Str("url", jobDetailsPage).Str("suite", opts.Suite).Str("platform", opts.PlatformName)
+	if opts.Framework == config.KindEspresso {
+		l.Str("device", opts.DeviceName).Str("name", opts.Name)
```

Comment:
@tianfeng92 can we add the `platformVersion` too?

Reply:
sure.